### PR TITLE
(1541) Fix inconsistencies with activity tabs and their ARIA states

### PR DESCRIPTION
**Pull request by leeky**
There were a few places where the ARIA selected hint didn't match the currently selected tab, so I'm using a bit of logic to tie the CSS classes and ARIA attributes together.

## Next steps

- [ ] Is an ADR required? An ADR should be added if this PR introduces a change to the architecture.
- [X] Is a changelog entry required? An entry should always be made in `CHANGELOG.md`, unless this PR is a small tweak which has no impact outside the development team.
- [ ] Do any environment variables need amending or adding?
- [ ] Have any changes to the XML been checked with the IATI validator? See [XML Validation](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/develop/doc/xml-validation.md)

### CODE REVIEW

File: app/views/staff/shared/activities/_tab_nav_item.html.haml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+- selected = controller_name.ends_with?(tab)
+%li{class: ["govuk-tabs__list-item", ("govuk-tabs__list-item--selected" if selected)] }
```
Comment (CristinaRO) [inline app/views/staff/shared/activities/_tab_nav_item.html.haml:2]: Tiny comment: would it be neater to add a `.compact` at the end of the array, or does the `nil` class not produce any cruft in the output at all?

**Review by CristinaRO — APPROVED** [review]
I like the views having a Railsier structure! I was always struggling to guess their place in the file tree. And good catch on the ARIA stuff.

File: app/views/staff/shared/activities/_tab_nav_item.html.haml (hunk 1 above)
Comment (leeky) [reply app/views/staff/shared/activities/_tab_nav_item.html.haml:2]: It actually swallows any `nil`s automatically :)

**Pull request merged**